src/Components/Search/SearchBar.tsx

```diff
@@ -159,7 +166,12 @@ export class SearchBar extends Component<Props, State> {
 
   renderInputComponent = inputProps => (
     <Box>
-      <Input style={{ width: "100%" }} {...inputProps} />
+      <Input
+        style={{ width: "100%" }}
+        innerRef={inputProps.ref}
+        {...inputProps}
+        ref={null}
```

Comment:
why explicitly setting null here?

Reply:
Without that you get an error/warning, causing the test to fail: `Warning: Input: 'ref' is not a prop. Trying to access it will result in 'undefined' being returned`

This is from https://github.com/moroshko/react-autosuggest/issues/469

> Debugging a custom input that is a `styled-component`, not necessarily needing the ref as you do, `ref={null}` was the key

Reply:
Hmm, yeah we might be running into some incompatibility stuff between styled-components 3 and 4. I'm going to finish this update today.